Description: Do not include <curl/types.h>
 libcurl4-gnutls-dev no longer comes with types.h. But this file was empty in
 previous versions and can be safely removed

Author: Vincent Legout <vincent@legout.info>

Index: gimmix/src/gimmix-covers.c
===================================================================
--- gimmix.orig/src/gimmix-covers.c	2011-08-02 21:26:53.253203823 +0200
+++ gimmix/src/gimmix-covers.c	2011-08-02 21:26:57.949203978 +0200
@@ -29,7 +29,6 @@
 #include <glib.h>
 #include <glib/gstdio.h>
 #include <curl/curl.h>
-#include <curl/types.h>
 #include <curl/easy.h>
 #include <nxml.h>
 #include <libxml/xmlreader.h>
Index: gimmix/src/gimmix-lyrics.c
===================================================================
--- gimmix.orig/src/gimmix-lyrics.c	2011-08-02 21:27:23.377204604 +0200
+++ gimmix/src/gimmix-lyrics.c	2011-08-02 21:27:26.705204751 +0200
@@ -30,7 +30,6 @@
 #include <nxml.h>
 #include <glib/gstdio.h>
 #include <curl/curl.h>
-#include <curl/types.h>
 #include <curl/easy.h>
 #include <libxml/xmlreader.h>
 #include "gimmix-lyrics.h"
